packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeLink/MT5MobileRedirectOption.tsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable camelcase */
```

Comment:
in wallets we are using camelcase for the sake consistency let's stick to using camelcase